### PR TITLE
Reduce size of test output

### PR DESCRIPTION
**Pull request by MaxThevenet**
We're trying to reduce the size of benchmarks, by reducing:
- resolution
- number of particles created
- fields and particle attributes written to file

it may require to increase the tolerance.

| test | before | after|
|--|--|--|
| qed_breit_wheeler_opt_depth_evolution  | 2.3 GB | 9.4 MB |
| Langmuir_multi  | 247 MB   | 34 MB |
| Langmuir_multi_nodal  | 247 MB   | 34 MB |
| Langmuir_multi_psatd  | 247 MB   | 34 MB |
| Langmuir_multi_psatd_nodal  | 247 MB   | 34 MB |
| Langmuir_x  | 61 MB   | 0 MB (deleted) |
| Langmuir_y  | 61 MB   | 0 MB (deleted) |
| Langmuir_z  | 61 MB   | 0 MB (deleted) |

Related to #855

### CODE REVIEW

File: Examples/Modules/qed/breit_wheeler/inputs_3d_optical_depth_evolution [hunk 1]
```diff
@@ -185,10 +195,10 @@ qed_bw.lookup_table_mode = "dummy_builtin"
...
 ### EXTERNAL E FIELD ### (3e15 * [-0.811107105653813 0.324442842261525 0.486664263392288] )
 particles.E_ext_particle_init_style = "constant"
-particles.E_external_particle = -2433321316961438 973328526784575 1459992790176863
+particles.E_external_particle = -486664263392286 194665705356914 291998558035372
 ####
 
 ### EXTERNAL B FIELD ### (1e7 * [0.28571429 0.42857143 0.85714286] )
 particles.B_ext_particle_init_style = "constant"
-particles.B_external_particle = 2857142.85714286 4285714.28571428 8571428.57142857
+particles.B_external_particle = 571429.7142856 857142.5714284 1714285.142857
```
Comment (lucafedeli88) [inline Examples/Modules/qed/breit_wheeler/inputs_3d_optical_depth_evolution]: I am not sure to understand this change. Could you please explain why you would like to change the field intensities here?

Comment (lucafedeli88) [reply Examples/Modules/qed/breit_wheeler/inputs_3d_optical_depth_evolution]: (ah, sorry... I failed to notice that the PR is still `[WIP]`)

**Comment by MaxThevenet** [comment]
Status update: Luca is doing to do modifications in `qed_breit_wheeler_opt_depth_evolution` in another PR. Once this is ready, I will delete the changes to `qed_breit_wheeler_opt_depth_evolution` in this PR and remove the [WIP] tag.

**Comment by MaxThevenet** [comment]
Actually should we merge this PR? Then we can use the new diags and coarsening to further reduce the size of output data.

**Pull request merged**